web/app/dashboard/dashboard.component.scss

```diff
@@ -1,29 +1,34 @@
 .fci-dashboard-header {
     background-color: #E1BEE7;
-    height: 448px;
-    margin-bottom: -112px;
     .fci-dashboard-welcome {
-        padding: 112px 0 52px 0;
+        padding: 75px 0 4px 0;
         .fci-dashboard-title {
-            color: rgba(0, 0, 0, 0.87);
             font-family: 'Google Sans', sans-serif;
-            font-size: 30px;
-            font-weight: 400;
+            color: rgba(0, 0, 0, 0.87);
+            font-size: 32px;
+            font-weight: 500;
             line-height: 40px;
-            margin-bottom: 8px;
+            margin-bottom: 12px;
```

Comment:
Would it make sense to use `em` instead of `px` for margins? I've always hated `px` and opt for `em` or `%`.